src/css/custom.css

```diff
@@ -22,28 +22,35 @@
   --ifm-heading-font-weight: bold;
   --ifm-line-height-base: 1.5;
   --ifm-color-primary: #5e33ff;
-  --ifm-color-primary-dark: #552ed9;
-  --ifm-color-primary-darker: #4c29b3;
-  --ifm-color-primary-darkest: #3d2194;
-  --ifm-color-primary-light: #7e5bff;
-  --ifm-color-primary-lighter: #9e83ff;
-  --ifm-color-primary-lightest: #beaaff;
+  --ifm-color-primary-dark: #4614ff;
+  --ifm-color-primary-darker: #3a05ff;
+  --ifm-color-primary-darkest: #2d00d6;
+  --ifm-color-primary-light: #7652ff;
+  --ifm-color-primary-lighter: #8261ff;
+  --ifm-color-primary-lightest: #a68fff;
+  --ifm-background-color: #ffffff;
   --ifm-code-font-size: 85%;
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.1);
 }
 
 /* For readability concerns, you should choose a lighter palette in dark mode. */
 [data-theme='dark'] {
-  --ifm-color-primary: #5e33ff;
-  --ifm-color-primary-dark: #552ed9;
-  --ifm-color-primary-darker: #4c29b3;
-  --ifm-color-primary-darkest: #3d2194;
-  --ifm-color-primary-light: #7e5bff;
-  --ifm-color-primary-lighter: #9e83ff;
-  --ifm-color-primary-lightest: #beaaff;
+  --ifm-color-primary: #a384ff;
+  --ifm-color-primary-dark: #865dff;
+  --ifm-color-primary-darker: #784aff;
+  --ifm-color-primary-darkest: #4c10ff;
+  --ifm-color-primary-light: #c0abff;
+  --ifm-color-primary-lighter: #cebeff;
+  --ifm-color-primary-lightest: #faf8ff;
+  --ifm-background-color: #1b1725 !important; /* Otherwise they are overwritten by html[data-theme='dark'] */
+  --ifm-background-surface-color: #32324a !important; /* Otherwise they are overwritten by html[data-theme='dark'] */
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.3);
 }
 
+footer {
+  --ifm-footer-background-color: #32324a !important;
```

Comment:
I understand this `!important` is needed for the same reason as those above.

Reply:
The important wasn't really needed, as I can choose a more specific class to override the value. Not sure which approach is best, TBH.

Thanks for pointing it out, anyway.

Reply:
I believe choosing a more specific selector is considered to be the best-practice. But either approach is fine with me.